include/cantera/kinetics/GasKinetics.h

```diff
@@ -67,6 +70,61 @@ class GasKinetics : public BulkKinetics
     //! reactions.
     virtual void update_rates_C();
 
+private:
+    //! @name Internal service methods
+    /*!
+     * These methods are for @internal use, and seek to avoid code duplication
+     * while evaluating terms used for rate constants, rates of progress, and
+     * their derivatives. Frequently used methods are defined in the header
+     * file and use raw data pointers to allow inlining and avoid overhead.
```

Comment:
I would guess that the performance benefits of implementing these in the header file is negligible for any function that operates on an array of size `nReactions`. Not to mention that `processFwdRateCoefficients` starts off by calling a couple of (supposedly) virtual functions.

Also, I'd suggest `protected` as the scope for these methods -- I think it's usually unnecessary to completely rule out the option of using methods in derived classes.

Reply:
Fair point!

Reply:
It looks like these are all still defined in the header -- was there a commit that didn't get pushed?

Reply:
No - I guess I misunderstood and just updated the wording :rofl: